example-cfgs/customer-settings.yaml

```diff
@@ -1,5 +1,5 @@
 # must be > previous customer settings version
-customer-settings-version: 1
-# secure_firmware_version: 0
-# nonsecure_firmware_version: 0
+customer-version: 1
+secure-firmware-version: 0
+nonsecure-firmware-version: 0
 rot-keys-status: ["Enabled", "Enabled", "Enabled", "Enabled"]
```

Comment:
Just FYI, the intent currently is that one config file format is good for everything, with subcommands only using/reading out what they need (and ideally TOML, the YAML thing was just an experiment): https://github.com/lpc55/lpc55-host/blob/734c929d853e68048a7445211658f8a652fdcf0e/src/bin/lpc55/main.rs#L66.